integration-tests/integration_test_suite_test.go

```diff
@@ -84,3 +85,21 @@ func deleteProperty(key string, properties map[string]any) map[string]any {
 	delete(properties, key)
 	return properties
 }
+
+func nthTerraformInvocationVars(p testframework.TerraformMock, n int) (map[string]any, error) {
```

Comment:
thoughts: we may want to move this method to the TerraformMock struct in `brokerpaktestframework`